### DISCUSSION TITLE
Adapting with the terminal width

### DISCUSSION THREAD

**Discussion by devallabharath** [simonmclean/tryptic#47] [Ideas]
Sometimes I use nvim tiled to the half screen. The 'Triptcyh' looks clumsy by squeezing all 3 windows(parent, primary, child), which makes the content unreadable. My idea is to remove the parent window when there is no enough width.

**Comment by simonmclean** [comment]
@devallabharath I've merged a change which replaces `options.column_widths` with `options.response_columns_widths`. This should achieve what you're looking for, and allow you to customise it
